python/lsst/validate/drp/validate.py

```diff
@@ -88,17 +92,23 @@ def loadAndMatchData(repo, visitDataIds,
     srcVis = SourceCatalog(newSchema)
 
     for vId in visitDataIds:
-        calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
-        calib.setThrowOnNegativeFlux(False)
+        try:
+            calib = afwImage.Calib(butler.get("calexp_md", vId, immediate=True))
+        except TypeError as te:
+            print(te)
+            continue
```

Comment:
Why would this throw a `TypeError`? (I'm not claiming it won't or can't, I'm just a little surprised.) If it happens, can we do something more informative than just printing the exception? (Even adding "Skipping to next data ID" might be useful, for e.g.).

Also, this code is added in 8fc82e0, which is fundamentally about a different issue (adding the no-throw decorator, below). Can we split it into its own commit with a message that explains why it's necessary?